src/types-and-traits.rst

```diff
@@ -2637,6 +2637,30 @@ A :t:`higher-ranked trait bound` is a :t:`bound` that specifies an infinite
 list of :t:`[bound]s` for all possible :t:`[lifetime]s` specified by the
 :s:`ForGenericParameterList`.
 
+:dp:`fls_J9DEsd06Ttu9`
+An :t:`implied bound` is a :t:`bound` that is not expressed in syntax, but is
+is the byproduct of relations between the :t:`[generic parameter]s` of
+:t:`[abstract data type]s` or :t:`[function]s`.
+
+:dp:`fls_IfHRxSasGAih`
+A :t:`reference` of the form ``&'a T``, where ``'a`` is a
+:t:`lifetime parameter` and ``T`` is a :t:`type parameter`, yields
+:t:`implied bound` ``T: 'a``.
+
+:dp:`fls_SMbHOyJgVeKB`
+A :t:`reference` of the form ``&'a &'b``, where ``'a`` and ``'b`` are
+:t:`[lifetime parameter]s``, yields :t:`implied bound` ``'a: 'b``.
```

Comment:
I think these are swapped – but then, this is equivalent to the previous rule, so it seems redundant

Reply:
Good catch. Anyway, I will merge the two rules.